### DISCUSSION TITLE
Vertical toolbar background missing in last release (0.19)

### DISCUSSION THREAD

**Discussion by ren2r** [tsujan/Kvantum#639] [General]
After update to the last release I noticed the background of the toolbar of the theme KVMojave or others that have background was missing when using the toolbar placed vertically.
I couldn't find if that was intended change, so I didn't reported it as issue without asking it first.

Here is a screenshot of how it looks now (version 0.19).
![image](https://user-images.githubusercontent.com/1781248/116268157-6eebbf80-a753-11eb-849c-d6068254edf1.png)

Here is a screenshot after doing a downgrade to version 0.18.0
![image](https://user-images.githubusercontent.com/1781248/116268328-96428c80-a753-11eb-9f3f-2e4d58aa0056.png)
Using a up to date archlinux system.

**Comment by ren2r** [comment]
I searched a bit and found the commit 3e406647276218b0e0ef30b5d3c8e965450e1209, I guess it was when the option was removed.
May I ask to have a option for enable it in settings or maybe another way to style vertical toolbar too from inside the config file throught another section (eg. `[VerticalToolbar]`)?

**Comment by tsujan** [comment]
Yes, the backgrounds of vertical toolbars were removed intentionally because they could look very unnatural under certain circumstances, e.g., when toolbar buttons were wide or when there were both vertical and horizontal toolbars.

I didn't add a key for it because I didn't want too many keys. However, I removed a few redundant keys; so, I can add `no_vertical_toolbar` with `true` as its default value.

**Comment by ren2r** [comment]
I undestand, it makes total sense to do this way.
I was just caught a litle by surprise because was making a custom theme based on kvmojave targeting vertical toolbar, as the screenshot bellow
![image](https://user-images.githubusercontent.com/1781248/116273474-2682d080-a758-11eb-8f86-9ed4a941b895.png)
And a option would be good enough. 
Apreciate your effort, thanks very much for this great software.

**Reply by tsujan** [reply]
I'll add a key for it to Kvantum and also a checkbox to Kvantum Manager soon.

**Reply by ren2r** [reply]
Thanks for your attention, just one last thought ...
wouldn't the `no_vertical_toolbar` key be kind of confusing? (leading the user to think that vertical toolbar would be disabled)
maybe something like `style_vertical_toolbar` set to `false` would be a bit more clearer.
By the way, Thanks so much one more time.

**Reply by tsujan** [reply]
> wouldn't the no_vertical_toolbar key be kind of confusing?

Yes, it is :) Here, I'd changed it to `style_vertical_toolbars` with a `false` value by default. Should add it to the doc and do some tests. Will tell you when it's ready.

Thanks for the suggestion!

**Reply by tsujan** [reply]
Done in https://github.com/tsujan/Kvantum/commit/3b979ad4367f0cc9088fb6964ad5f2ae9800a220

This screenshot shows the new option (belonging to "Hacks"):

![new_key](https://user-images.githubusercontent.com/981076/116297630-b5bbd280-a7b0-11eb-8832-a0c49664f359.png)

Thanks again for your comments and suggestions!

**Reply by ren2r** [reply]
I just tested it and it's working perfectly.
Thank you for your effort, thank you very much
